test/e2e/annotations/disableaccesslog.go

```diff
@@ -19,10 +19,11 @@ package annotations
 import (
 	"strings"
 
-	"github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/v2"
 
-	"k8s.io/ingress-nginx/test/e2e/framework"
 	"net/http"
+
+	"k8s.io/ingress-nginx/test/e2e/framework"
```

Comment:
The order of importing packages needs to be adjusted

Reply:
done